src/game-board/game-board.html

```diff
@@ -60,15 +60,24 @@
         //1 to avoid an out of bounds exception.
         var stackCardRank = rankList.indexOf(stack[stack.length-1].rank);
         //Simple comparison here. If the the card we want to place is a rank lower than the card on top of the stack (suit doesn't matter here)
-        //Then we return true. Else we return false. 
-        if(cardRank == stackCardRank - 1){
-          return true;
-        } else{
-          return false;
+        //Then we return true. Else we return false.
+        if(isFoundation == true){
+          if(cardRank == stackCardRank - 1){
+            return true;
```

Comment:
Unless I missed it elsewhere we also need to check to make sure that the suit is the same for foundation stacks.